.gitignore

```
@@ -33,6 +33,11 @@ yarn-error.log*
 # vercel
 .vercel
 
+# docs
+
+/docs/.next
```

I guess we could make it generic?

```suggestion
.next
```